tools/travis/run_ppa_autopkgtests.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright (C) 2015-2017 Canonical Ltd
+# Copyright (C) 2017 Canonical Ltd
```

Comment:
Is this change on purpose?

Reply:
Yes, the 2015 was a copy and paste mistake. This is a file we added this year.